snakefile.py

```diff
@@ -522,6 +549,38 @@ def render_qc_report_params(wildcards, input, output = None, threads = None, res
             for name in generated:
                 logger.info("  - {}".format(name))
 
+
+# ANNOT: If the any of the databases is not present, download it to the location
+# specified in the settings file.
+rule download_kraken_db:
+    output: directory(KRAKEN_DB)
```

Comment:
Should these be marked as `ancient`?  Either here or wherever they are used?

Reply:
Not sure about that... Ancient is only used to denote files that should always be assumed to be older than outputs and will therefore not trigger a rerun if they are updated. But if the databases are already there, they are older, if they are newly downloaded they will also already be older. However if the databases are updated, we do want them to trigger a rerun, so I don't see why we would need an ancient tag there. This is the first time I am thinking about the concept of the ancient tag though...